Packs/Core/Playbooks/playbook-WildFire_Malware.yml

```diff
@@ -1727,7 +1758,12 @@ inputs:
   value:
     complex:
       root: alert
-      accessor: initiatorsha256
+      transformers:
+      - operator: DT
+        args:
+          dt:
+            value:
+              simple: .=pickvalue(val);function pickvalue(x){if(x.initiatorsha256){return x.initiatorsha256} else {return x.filesha256}}
   required: false
   description: The SHA256 hash of the suspected file.
```

Comment:
Please add here that it will based on the logic you mentioned above with the DT

Reply:
Done